controllers/nutanixcluster_controller.go

```diff
@@ -181,6 +182,10 @@ func (r *NutanixClusterReconciler) reconcileDelete(rctx *nctx.ClusterContext) (r
 
 func (r *NutanixClusterReconciler) reconcileNormal(rctx *nctx.ClusterContext) (reconcile.Result, error) {
 
+	if rctx.NutanixCluster.Status.FailureReason != nil || rctx.NutanixCluster.Status.FailureMessage != nil {
```

Comment:
what is the way to re-trigger reconciliation in this case then?

Reply:
also what about errors during delete reconciliation?

Reply:
FailureReason and FailureMessage are only set when a Fatal error occurs hence we stop the loop in this case.